learning_paths/api/v1/serializers.py

```diff
@@ -74,3 +74,14 @@ class Meta:
 class LearningPathProgressSerializer(serializers.Serializer):
     learning_path_id = serializers.UUIDField()
     aggregate_progress = serializers.FloatField()
+
+
+class LearningPathGradeSerializer(serializers.Serializer):
+    """
+    Serializer for learning path grade.
+    """
+
+    learning_path_id = serializers.UUIDField()
+    is_completion_threshold_met = serializers.BooleanField()
+    aggregate_grade = serializers.FloatField()
```

Comment:
I think `aggregate_something` (used as noun) is correct, since we want to emphasize the final result. But if you think we should be highlighting the action, then I can change it. What are your thoughts?

Reply:
@pkulkark, let's just use the terms `grade` and `progress` to keep things simple.

Reply:
Sounds good. Updated them.